src/desktop/src/ui/views/exchanges/MoonPay/ReviewPurchase.js

```diff
@@ -79,10 +81,24 @@ class ReviewPurchase extends React.PureComponent {
     }
 
     render() {
-        const { history, isCreatingTransaction, createTransaction, arePurchasesSuspended, children, t } = this.props;
+        const {
+            history,
+            isCreatingTransaction,
+            createTransaction,
+            arePurchasesSuspended,
+            children,
+            t,
+            hasAnyPaymentCards,
+        } = this.props;
 
         return (
-            <form onSubmit={() => arePurchasesSuspended ? history.push('/exchanges/moonpay/purchase-suspended-warning'): createTransaction}>
+            <form
+                onSubmit={() =>
+                    arePurchasesSuspended
+                        ? history.push('/exchanges/moonpay/purchase-suspended-warning')
+                        : createTransaction
```

Comment:
Why aren't we calling `createTransaction` here? If purchases aren't suspended, form submission won't do anything.